### DISCUSSION TITLE
[Hubs Cloud]Links not clickable on mobile

### DISCUSSION THREAD

**Discussion by ulitmate** [mozilla/hubs#5355] [Hubs Cloud Questions]
**Description**
A handful of users have tried to visit some of our worlds via their mobile devices but have found it difficult to click on the links that allow them to join those worlds. We noticed this issue only happens when trying to enter some world via mobile devices and not via PC or VR

**To Reproduce**
1. Go to 'www.rdland.me' via mobile device
![Screenshot_20220406-172728](https://user-images.githubusercontent.com/23181034/162426365-9ebbeda5-f281-4f6e-9985-f1a4d74738a5.png)

2. Click on 'on any of these portals '

**Expected behavior**
You wont be able to enter the world  stellArt and ropeland
![IMG_0756](https://user-images.githubusercontent.com/23181034/162426173-22acb525-7ce6-4c08-8331-c1b115a2979c.png)

**Comment by matthewbcool** [comment]
Thanks for reporting, I'm able to reproduce the issue on your custom client. android (samsung galaxy S5)

I was unable to reproduce on my own hubs cloud. android (samsung galaxy S5)
Steps I tried:
- Made a scene with a link to another room (using the link element)
- entered the scene on my mobile phone
- clicked 'Visit Room'
- I was directed to the room I linked to as expected

Can you give us more details about how you are making these links in the scene? Did you use the blender exporter? Have you made any relevant code changes? 

My guess is the difficulty in clicking the link is related to changes or scene design since my room link function as expected on hubs cloud. (at least for android). Please include some more details like device info - maybe this is an ios issue?

Thanks again for the report!
